src/app/public/modules/page/page.component.html

```diff
@@ -0,0 +1,3 @@
+<main class="sky-theme-default">
+  <ng-content></ng-content>
+</main>
```

Comment:
Some more food for thought around `main`:
> For Internet Explorer 11 and lower, it's suggested that an ARIA role of "main" be added to the `<main>` element to ensure it is accessible (screen readers like JAWS, used in combination with older versions of Internet Explorer, understand the semantic meaning of the `<main>` element when this role attribute is included).
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main